framework/test/unit/engine/consensus/certificate_generation/commit_pool.spec.ts

```diff
@@ -100,10 +100,6 @@ describe('CommitPool', () => {
 		});
 	});
 
-	describe('constructor', () => {
-		it.todo('');
-	});
-
 	describe('job', () => {
```

Comment:
I think it'd also a good idea to improve `describe('function_name')` with the functions names in commit_pool (e.g. those underscores) in this PR